tmt/base.py

```diff
@@ -2576,6 +2576,7 @@ class Result:
         'info': 'blue',
         'warn': 'yellow',
         'error': 'magenta',
+        'custom': 'black'
```

Comment:
Is 'custom' a valid value for Result class? IMO no.

Reply:
It looks it doesn't need to be there. Removed.